.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Link
+<!-- A link to the shard's source -->
+
+## Description
+<!-- A short description of what the shard does -->
+
+## Checklist
+* [ ] - Shard is at least 30 days old.
+* [ ] - Shard has CI implemented.
```

Comment:
Maybe add in something about including a badge in the readme?

Reply:
I would leave it as simple as is. We already had such a template but decided to remove it because of a long checklist.

Reply:
Was thinking of including it with this bullet point, versus its own line.

`Shard has CI implemented with CI badge in the readme`?

I don't feel strongly about it either way.

Reply:
Since we have a script to check CI status, badge in the readme is not that important anymore. From the reviewer's standpoint, the latest commit checks can be observerd. What do you think? 

![image](https://user-images.githubusercontent.com/3624712/68088094-3fad0880-fe64-11e9-9f55-9dc28a71f507.png)

Reply:
Sorry, I think I wasn't clear enough.  I was suggesting that the shard being added have a CI badge in _their_ readme.

Reply:
I understood it right. I meant, I don't want to force the shards to add badges to their readme, because we don't need it. As a reviewer, we can just check their latest commit (screen was just an example).